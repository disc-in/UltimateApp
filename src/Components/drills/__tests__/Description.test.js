import React from 'react';
import { render } from '@testing-library/react-native';

import { createDrill } from '../../../Fixtures/TestFixtures';

import Description from '../Description';

describe('<Description />', () => {
  it('renders correctly', () => {
    const drill = createDrill();

    const { toJSON } = render(<Description drill={drill} />);
    expect(toJSON()).toMatchSnapshot();
  });
  it('renders correctly as minimal', () => {
    const drill = createDrill();

    const { toJSON } = render(<Description drill={drill} minimal />);
    expect(toJSON()).toMatchSnapshot();
  });
});
