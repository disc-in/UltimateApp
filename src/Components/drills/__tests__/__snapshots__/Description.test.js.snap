// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Description /> renders correctly 1`] = `
<View
  style={
    Object {
      "paddingBottom": 10,
    }
  }
>
  <View
    style={
      Object {
        "borderBottomColor": "#e5e5e5",
        "borderBottomWidth": 1,
      }
    }
  />
  <View
    style={
      Object {
        "padding": 20,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#232b2b",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 5,
            "textAlign": "center",
          },
          undefined,
        ]
      }
    >
      Good for
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#A6A6A6",
            "fontSize": 16,
            "textAlign": "center",
          },
          undefined,
        ]
      }
    >
      Full body
    </Text>
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#e5e5e5",
        "borderBottomWidth": 1,
      }
    }
  />
  <View
    style={
      Object {
        "padding": 20,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#232b2b",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 5,
            "textAlign": "center",
          },
          undefined,
        ]
      }
    >
      Equipment
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#A6A6A6",
            "fontSize": 16,
            "textAlign": "center",
          },
          undefined,
        ]
      }
    >
      Equipment needs for the drill
    </Text>
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#e5e5e5",
        "borderBottomWidth": 1,
      }
    }
  />
  <View
    style={
      Object {
        "padding": 20,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#232b2b",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 5,
            "textAlign": "center",
          },
          undefined,
        ]
      }
    >
      Description
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#A6A6A6",
            "fontSize": 16,
            "textAlign": "center",
          },
          undefined,
        ]
      }
    >
      Description of the drill
    </Text>
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#e5e5e5",
        "borderBottomWidth": 1,
      }
    }
  />
</View>
`;

exports[`<Description /> renders correctly as minimal 1`] = `
<View
  style={
    Object {
      "paddingBottom": 10,
    }
  }
>
  <View
    style={
      Object {
        "borderBottomColor": "#e5e5e5",
        "borderBottomWidth": 1,
      }
    }
  />
  <View
    style={
      Object {
        "padding": 10,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#232b2b",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 5,
            "textAlign": "center",
          },
          Object {
            "textAlign": "left",
          },
        ]
      }
    >
      Equipment
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#A6A6A6",
            "fontSize": 16,
            "textAlign": "center",
          },
          Object {
            "textAlign": "left",
          },
        ]
      }
    >
      Equipment needs for the drill
    </Text>
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#e5e5e5",
        "borderBottomWidth": 1,
      }
    }
  />
  <View
    style={
      Object {
        "padding": 10,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#232b2b",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 5,
            "textAlign": "center",
          },
          Object {
            "textAlign": "left",
          },
        ]
      }
    >
      Description
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#A6A6A6",
            "fontSize": 16,
            "textAlign": "center",
          },
          Object {
            "textAlign": "left",
          },
        ]
      }
    >
      Description of the drill
    </Text>
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#e5e5e5",
        "borderBottomWidth": 1,
      }
    }
  />
</View>
`;
